### PR TITLE
Updates Several RN Packages

### PR DESCRIPTION
**Pull request by redreceipt**
## DESCRIPTION

Updates which were needed to get CF working with Android on latest RN version.

- [x] "react-native-image-picker": "0.28.0",
- [x] "react-native-maps": "^0.24.0",
- [x] "react-native-music-control": "0.10.0",
- [x] "react-native-screens": "1.0.0-alpha.22"
- [x] "react-native-video": "4.4.1",

## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [#656]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

**Comment by redreceipt** [comment]
Error during JS bundle of rn-music-control:

```
apolloschurchapp: error: bundling failed: Error: While trying to resolve module `react-native-music-control` from file `/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/src/ui/MediaPlayer/MusicControls.js`, the package `/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/react-native-music-control/package.json` was successfully found. However, this package itself specifies a `main` module field that could not be resolved (`/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/react-native-music-control/lib/index.js`. Indeed, none of these files exist:
apolloschurchapp:   * `/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/react-native-music-control/lib/index.js(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
apolloschurchapp:   * `/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/react-native-music-control/lib/index.js/index(.native||.ios.js|.native.js|.js|.ios.json|.native.json|.json|.ios.ts|.native.ts|.ts|.ios.tsx|.native.tsx|.tsx)`
apolloschurchapp:     at ResolutionRequest.resolveDependency (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/node-haste/DependencyGraph/ResolutionRequest.js:65:15)
apolloschurchapp:     at DependencyGraph.resolveDependency (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/node-haste/DependencyGraph.js:273:16)
apolloschurchapp:     at Object.resolve (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/lib/transformHelpers.js:261:42)
apolloschurchapp:     at dependencies.map.result (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:391:31)
apolloschurchapp:     at Array.map (<anonymous>)
apolloschurchapp:     at resolveDependencies (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:388:18)
apolloschurchapp:     at /Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:261:33
apolloschurchapp:     at Generator.next (<anonymous>)
apolloschurchapp:     at asyncGeneratorStep (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:87:24)
apolloschurchapp:     at _next (/Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/node_modules/metro/src/DeltaBundler/traverseDependencies.js:107:9)
apolloschurchapp:  BUNDLE  [ios, dev] ./index.js ▓▓▓▓▓▓▓░░░░░░░░░ 46.4% (1328/1990), failed.
apolloschurchapp: ::1 - - [26/Apr/2019:20:10:08 +0000] "GET /index.bundle?platform=ios&dev=true&minify=false HTTP/1.1" 200 - "-" "apolloschurchapp/8 CFNetwork/978.0.7 Darwin/18.2.0"
```

**Comment by redreceipt** [comment]
trying to update screens fails on the Android build:

```
apolloschurchapp: > Task :app:compileDebugJavaWithJavac FAILED
apolloschurchapp: /Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/android/app/src/main/java/com/apolloschurchapp/MainApplication.java:12: error: cannot find symbol
apolloschurchapp: import com.swmansion.rnscreens.RNScreenPackage;
apolloschurchapp:                               ^
apolloschurchapp:   symbol:   class RNScreenPackage
apolloschurchapp:   location: package com.swmansion.rnscreens
apolloschurchapp: /Users/michael.neeley/Documents/Projects/apollos-prototype/packages/apolloschurchapp/android/app/src/main/java/com/apolloschurchapp/MainApplication.java:46: error: cannot find symbol
apolloschurchapp:             new RNScreenPackage(),
apolloschurchapp:                 ^
apolloschurchapp:   symbol: class RNScreenPackage
apolloschurchapp: 2 errors
apolloschurchapp: FAILURE: Build failed with an exception.
apolloschurchapp: * What went wrong:
apolloschurchapp: Execution failed for task ':app:compileDebugJavaWithJavac'.
apolloschurchapp: > Compilation failed; see the compiler error output for details.
```

**Comment by redreceipt** [comment]
cc @vinnyjth

**Comment by vinnyjth** [comment]
@redreceipt The `react-native-screens` guy, he renamed the package to "Screens" on Android, so make sure the plurality is correct in the `.java` files. 

The `music-control-one` is b/c the guy published an updated version for me, but forgot to include the JS. You can actually jump to `0.10.2`

**Comment by redreceipt** [comment]
@vinnyjth this one is ready except for the same thing in the 0.8.3 release PR, there's a snapshot that changed very slightly for some reason that is causing the tests to fail. I think it's probably safe to update but wanted you to make that call.

**Comment by redreceipt** [comment]
blocked on #662

**Comment by vinnyjth** [comment]
@conrad-vanl If you have time today, giving this a thorough review will unlock a lot of value.

File: packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap [hunk 1]
```diff
@@ -11911,7 +11912,7 @@ exports[`Test the Discover Component Query Should retrieve the Content Channel F
           <View
             style={
               Object {
-                "backgroundColor": "#F7F7F7",
+                "backgroundColor": "#FFF",
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap]: These snapshot changes have me concerned...any idea why these colors changed?

File: packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap [hunk 2]
```diff
@@ -11985,14 +11975,15 @@ exports[`Test the Discover Component Query Should retrieve the Content Channel F
...
+                      accessible={false}
                       allowFontScaling={true}
                       numberOfLines={1}
                       onLayout={[Function]}
                       style={
                         Object {
                           "color": "rgba(0, 0, 0, .9)",
                           "fontSize": 17,
-                          "fontWeight": "700",
+                          "fontWeight": "600",
```
Comment (conrad-vanl) [inline packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap]: Same as above...concerned that these styles are changing.

File: packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap (hunk 1 above)
Comment (redreceipt) [reply packages/apolloschurchapp/src/tabs/discover/__snapshots__/discover.tests.js.snap]: I saw that. No idea. But I did inspect the discover tab and it doesn't look strange so I don't know what else to do.
